sdk/src/client/node_api/core/routes.rs

```diff
@@ -263,7 +263,7 @@ impl ClientInner {
 
     // UTXO routes.
 
-    /// Finds an output, as JSON, by its OutputId (TransactionId + output_index).
+    /// Finds an output, as JSON, by its identifier.
```

Comment:
Pawel's comment made me also think about this doc ... and honestly ... I still find it confusing, but maybe it's just me. 
Why JSON? We are returning an actual type, not a JSON string (even if that type can be serialized into a JSON string) :thinking:

Reply:
Because what is being found is JSON, not an actual type. This sentence says nothing about the returned type. But I can change it anyway.

Reply:
Done